scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -1613,3 +1613,246 @@ cdef class DiscreteAliasUrn(Method):
+    of `r` increase the class of distributions for which the method works at
+    the expense of a higher rejection constant. For computational reasons,
+    ``r=1`` should be used if possible.
+
+    Parameters
+    ----------
+    dist : object
+        An instance of a class with ``pdf`` method.
+
+        * ``pdf``: PDF of the distribution. The signature of the PDF is
+          expected to be: ``def pdf(self, x: float) -> float``. i.e.
+          the PDF should accept a Python float and the parameters and
+          return a Python float. It doesn't need to integrate to 1 i.e.
+          the PDF doesn't need to be normalized.
+
+    center : float, optional
+        Approximate location of the mode or the mean of the distribution.
+        This location provides some information about the main part of the
+        PDF and is used to avoid numerical problems. Default is ``None``.
+
+    domain : list or tuple of length 2, optional
+        The support of the distribution.
+        Default is ``None``. When ``None``:
+
+        * If a ``support`` method is provided by the distribution object
+          `dist`, it is used to set the domain of the distribution.
+        * Otherwise the support is assumed to be :math:`(-\infty, \infty)`.
+
+    r : float, optional
+        Set parameter ``r > 0`` for generalized Ratio-Of-Uniforms method.
+        The default is 1.0.
+
+    u_min: float
+        Sets the left boundary of the bounding rectangle, see details under
+        Notes. If necessary, consider computing the bound numerically using
+        methods in `scipy.optimize`. Important: The bound has to be
+        provided for ``dist.pdf(x - center)``.
+
+    u_max: float
+        Sets the right boundary of the bounding rectangle. Sets the left
+        boundary of the bounding rectangle, see details under
+        Notes. If necessary, consider computing the bound numerically using
+        methods in `scipy.optimize`. Important: The bound has to be
+        provided for ``dist.pdf(x - center)``.
+
+    v_max: float
+        Sets the upper boundary of the bounding rectangle. If the mode of the
+        pdf is known, it can be easily computed, see Notes for details.
+        Otherwise consider computing the bound numerically using
+        methods in `scipy.optimize`.
+
+    random_state : {None, int, `numpy.random.Generator`,
+                        `numpy.random.RandomState`}, optional
+
+        A NumPy random number generator or seed for the underlying NumPy random
+        number generator used to generate the stream of uniform random numbers.
+        If `random_state` is None (or `np.random`), the `numpy.random.RandomState`
+        singleton is used.
+        If `random_state` is an int, a new ``RandomState`` instance is used,
+        seeded with `random_state`.
+        If `random_state` is already a ``Generator`` or ``RandomState`` instance
+        then that instance is used.
+
+    Notes
+    -----
```

Comment:
I won't comment specifically about uses of single/double backticks and how equations are represented, but please consider whether symbols are appearing as you intend.